pkg/api/customization/catalog/validator.go

```diff
@@ -15,7 +16,17 @@ var (
 	controlEncoded = regexp.MustCompile("%[0-1][0-9,a-f,A-F]")
 )
 
-func Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+func (w Wrapper) Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+	if request.Method == http.MethodPut {
```

Comment:
Let's move this into a store update method then use a helper so this logic isn't duplicated